indico/web/templates/forms/multiple_items_widget.html

```diff
@@ -21,9 +21,13 @@
             </thead>
             <tbody></tbody>
         </table>
-        <button type="button" id="{{ field.id }}-add-button" class="js-add-row i-button icon-plus">
-            {%- trans %}Add{% endtrans -%}
-        </button>
+        {% block action_buttons %}
+        <div class="action_buttons">
+            <button type="button" id="{{ field.id }}-add-button" class="js-add-row i-button icon-plus">
+                {%- trans %}Add{% endtrans -%}
+            </button>
+        </div>
```

Comment:
- Indent the contents of the block
- Do you really need that div? If you already use the block to add custom stuff you can just wrap the `{{ super() }}` call in the div in your custom code.

Reply:
You're correct, I don't need that div if I summon the super inside our div, thanks